models/public/robust-video-matting/README.md

```diff
@@ -0,0 +1,226 @@
+# robust-video-matting
+
+## Use Case and High-Level Description
+
+The `robust-video-matting` model is a robust high-resolution human video matting method that
```

Comment:
I would like to avoid such loud words as `new state of the art`, who knows how long model will be part of OMZ :)

Reply:
Loud words were removed 😄.